core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -18,16 +18,40 @@
       GLOBALS.INTERACTION_SPECS = JSON.parse(
         '{{INTERACTION_SPECS|js_string}}');
     </script>
+
   {% endif %}
 
   {{dependencies_html}}
+  <style>
```

Comment:
@himanshu-dixit  Maybe you can add your inline css to components/top_navigation_bar/top_navigation_bar_directive.html instead of exploration_player.html? I think this is the file where the changes are taking place you are intended to do.